benchmarks/distributed/rpc/rl/README.md

```diff
@@ -0,0 +1,64 @@
+# Distributed RPC Benchmark
```

Comment:
This is not a full-fledged RPC benchmark. Instead, it only measures a very specific use case. Let's be clear on that.